test/functional-portable/datastoresrp/cloud/resources/microsoftsql_test.go

```diff
@@ -16,66 +16,4 @@ limitations under the License.
 
 package resource_test
 
-import (
-	"os"
-	"testing"
-
-	"github.com/radius-project/radius/test/rp"
-	"github.com/radius-project/radius/test/step"
-	"github.com/radius-project/radius/test/testutil"
-	"github.com/radius-project/radius/test/validation"
-)
-
-func Test_MicrosoftSQL_Manual(t *testing.T) {
-	template := "testdata/datastoresrp-resources-microsoft-sql.bicep"
-	name := "dsrp-resources-microsoft-sql"
-
-	var adminUsername, adminPassword string
-
-	if os.Getenv("AZURE_MSSQL_RESOURCE_ID") == "" {
-		t.Error("AZURE_MSSQL_RESOURCE_ID environment variable must be set to run this test.")
-	}
-	if os.Getenv("AZURE_MSSQL_DATABASE") == "" || os.Getenv("AZURE_MSSQL_SERVER") == "" {
-		t.Error("AZURE_MSSQL_DATABASE and AZURE_MSSQL_SERVER environment variable must be set to run this test.")
-	}
-	if os.Getenv("AZURE_MSSQL_USERNAME") != "" && os.Getenv("AZURE_MSSQL_PASSWORD") != "" {
-		adminUsername = "adminUsername=" + os.Getenv("AZURE_MSSQL_USERNAME")
-		adminPassword = "adminPassword=" + os.Getenv("AZURE_MSSQL_PASSWORD")
-	} else {
-		t.Error("AZURE_MSSQL_USERNAME and AZURE_MSSQL_PASSWORD environment variable must be set to run this test.")
-	}
-
-	mssqlresourceid := "mssqlresourceid=" + os.Getenv("AZURE_MSSQL_RESOURCE_ID")
-	sqlDatabse := "database=" + os.Getenv("AZURE_MSSQL_DATABASE")
-	sqlServer := "server=" + os.Getenv("AZURE_MSSQL_SERVER")
-	appNamespace := "default-dsrp-resources-microsoft-sql"
-
-	test := rp.NewRPTest(t, name, []rp.TestStep{
-		{
-			Executor: step.NewDeployExecutor(template, testutil.GetMagpieImage(), mssqlresourceid, adminUsername, adminPassword, sqlDatabse, sqlServer),
-			RPResources: &validation.RPResourceSet{
-				Resources: []validation.RPResource{
-					{
-						Name: name,
-						Type: validation.ApplicationsResource,
-					},
-					{
-						Name: "mssql-app-ctnr",
-						Type: validation.ContainersResource,
-						App:  name,
-					},
-				},
-			},
-			K8sObjects: &validation.K8sObjectSet{
-				Namespaces: map[string][]validation.K8sObject{
-					appNamespace: {
-						validation.NewK8sPodForResource(name, "mssql-app-ctnr"),
-					},
-				},
-			},
-		},
-	})
-
-	test.RequiredFeatures = []rp.RequiredFeature{rp.FeatureAzure}
-	test.Test(t)
-}
+// TODO: This test will be deleted in a followup PR
```

Comment:
Keeping this test for now so that the functional tests pass. This will be deleted in a followup PR with additional updates to the `datastoresrp-cloud` make targets